### ISSUE TITLE
Negative flee!

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/diceydungeons.com#533]
Ok, this one's weird:

```
so I was trying to figure out, in the level editor, exactly what item it was causing copycat fights to crash
```

![fleecoun](https://user-images.githubusercontent.com/2874150/46431290-50f88280-c743-11e8-87f1-93b64126b06c.png)

```
......does that Flee counter say "-1"???
.....the very next turn after that, it said "-3"
so I guess at some odd-numbered point, instead of decrementing by 1, it decremented by 2
```

**Comment by TerryCavanagh** [comment]
Haven't seen this in a while. I think it's fixed?

**Comment by Ereza** [comment]
@TerryCavanagh, I have some bad news: the issue is still there.

I was watching a gameplay on Twitch and the streamer ended up with a "Flee in -1". These are the steps followed:

- Start a "Double Rubble" (Inventor episode 2) game.
- At a certain point, flee from a battle at turn 2.
- Start a battle against the same enemy:
  - Turn 1: "Flee in 1"
  - Turn 2: Flee is available (don't use it)
  - Turn 3: Flee is available (don't use it)
  - Win battle
- Start a battle against another enemy:
  - Turn 1: "Flee in 1"
  - Turn 2: Flee is available, use it
- Start a battle against the same enemy:
  - Turn 1: "Flee in 2"
  - Turn 2: **"Flee in 0" (!!!)**
  - Turn 3: Flee is available (don't use it)
  - Turn 4: Flee is available (don't use it)
  - Turn 5: Flee is available (don't use it)
  - Turn 6: Flee is available, use it
- Start a battle against the same enemy:
  - Turn 1: "Flee in 3"
  - Turn 2: **"Flee in 1" (!!!)**
  - Turn 3: **"Flee in -1" (!!!)**

Here are some screenshots:

![flee_0](https://user-images.githubusercontent.com/285008/99138871-4011df00-2634-11eb-89de-b0171f33e15c.png)
![flee_minus1](https://user-images.githubusercontent.com/285008/99138873-4607c000-2634-11eb-95f4-5d3ea63886e9.png)

**Comment by TerryCavanagh** [comment]
Ah! Thanks for the repo! I'll investigate and fix :)

**Comment by TerryCavanagh** [comment]
Ok, figured this out! Fixed for v1.11.
